SU-HSDO/suhumsci
# STN-504: Increase Traditional Font Size

# READY FOR REVIEW

## Summary
The main goal of this PR is to update the body font in the Traditional theme to `1.8rem //18px`. Increasing the body font also required that we adjust a few other elements on the theme:

- increase heading in ptype-hs-row to 22px in the traditional theme
- set field-content font-size to 18px in traditional theme
- decrease 3rd column font size in the traditional theme
- increase hs-caption and hs-credits font size in traditional theme
- create hb-credits mixin and use for figcaption
- increase card title font-size on mobile for traditional theme

⚠️  These changes should only affect the Traditional theme
⚠️ This work is blocking STN-348

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. All tests should pass
1. Confirm that there are no changes to the Colorful theme
1. Confirm that the font-size of the Traditional theme is now 1.8rem (18px)
1. Confirm the following:
    - `h2`/Titles in `.hb-grid` were increased to 22px
        ![Home_Page___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/90550894-928b4e80-e15e-11ea-90d1-9b2dc1ce43ab.png)
    - `.hb-card__subcontent-item` was increased to 18px
        ![Events___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/90550985-b51d6780-e15e-11ea-9cf2-602692113afc.png)
    - `.hs-caption` and `.hs-credits` were increased to 16px 
        ![QA_text_area___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/90551109-daaa7100-e15e-11ea-9474-e8193e92b886.png)
        - The `figcaption` was updated to use the `.hs-caption` styles
            ![QA_Image_Styles___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/90551180-f746a900-e15e-11ea-9d95-357471da9c16.png)
    - The font styles in the right most column of 3 column layouts were updated to:
        ```SCSS
        font-size: 16px;
        line-height: 127%;
        ```
        ![Ran_Abramitzky___QA_Sandbox__Stage_for_Colorful__Dev_for_Traditional_](https://user-images.githubusercontent.com/2486846/90551355-2f4dec00-e15f-11ea-88c1-f4f7b300f3a0.png)




## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
